src/lib/Scenes/Artwork/Components/ArtworkAvailability.tsx

```diff
@@ -12,7 +12,7 @@ export class ArtworkAvailability extends React.Component<ArtworkAvailabilityProp
   render() {
     return (
       <Box>
-        <Sans size="4t" weight="medium">
+        <Sans size="3t" weight="medium">
```

Comment:
Not sure if I'm looking at something outdated, but my zeplin spec has this being `4t`.
![image](https://user-images.githubusercontent.com/2081340/60889714-a8be9f80-a227-11e9-9757-037614610593.png)

Reply:
Ah good catch thanks